Portal/src/Datahub.Portal/Pages/Workspace/Dashboard/AppServiceCard.razor

```diff
@@ -0,0 +1,48 @@
+﻿@using Datahub.Core.Model.Projects
+
+<MudPaper Outlined Class="pa-4">
+    <MudStack>
+        <MudStack Row AlignItems="AlignItems.Center">
+            <MudIcon Icon="@SidebarIcons.WebApp" Size="Size.Medium"/>
+            <MudText Typo="Typo.h3">@Localizer["Web Application"]</MudText>
+            <MudSpacer/>
+
+        </MudStack>
+        <MudText>
+            @Localizer["Navigate to the Azure App Service to view your web application, where you can serve interactive and user-friendly dashboards."]
+        </MudText>
+        <MudElement HtmlTag="div">
+            <MudButton
+                Href="@GetWebAppUrl()"
+                Target="_blank"
+                Color="Color.Primary"
+                Variant="@Variant.Text"
+                EndIcon="@Icons.Material.Outlined.OpenInNew">
```

Comment:
I've been moving over to the font awesome fa-light version, check out the file `SidebarIcons` for some static references.

Bit of a misnomer as originally it was for the sidebar, but having all the icon references in one spot for consistency is nice